app/frontend/src/service/conversations.ts

```diff
@@ -50,3 +51,9 @@ export async function sendMessage(groupChatId: number, text: string) {
   req.setText(text);
   return await client.conversations.sendMessage(req);
 }
+
+export async function leaveGroupChat(groupChatId: number) {
+  const req = new LeaveGroupChatReq();
+  req.setGroupChatId(groupChatId);
+  return await client.conversations.leaveGroupChat(req);
+}
```

Comment:
```suggestion
export function leaveGroupChat(groupChatId: number) {
  const req = new LeaveGroupChatReq();
  req.setGroupChatId(groupChatId);
  return client.conversations.leaveGroupChat(req);
}
```
I think you can return Promise and let the function caller await the result.